scss/civicrm/common/_modals.scss

```diff
@@ -150,6 +167,15 @@
         .ui-button-text {
           font-size: $font-size-base;
         }
+
+        &.ui-button-text-icon-primary {
+          padding-left: $crm-btn-padding-base-horizontal + $ui-icon-margin;
+        }
+
+        .ui-button-icon-primary.ui-icon {
+          margin-top: -$ui-icon-margin;
```

Comment:
Is there a way to avoid negative margins?

Reply:
jQuery UI adds a default -8px margin-top to icons:

![untitled](https://user-images.githubusercontent.com/1642119/31032759-cc3bac72-a509-11e7-9644-fdff0259ffb6.png)

Here I'm changing it to -7px to better align the icon with the text, otherwise it looks out of sync:

### Before
![download 60](https://user-images.githubusercontent.com/1642119/31032868-45c4dc8a-a50a-11e7-8935-69c10b5b6173.png)

### After
![download 58](https://user-images.githubusercontent.com/1642119/31032873-4aa1fbca-a50a-11e7-8547-939fd53604bd.png)
(yes, yes, almost unnoticeable)